base_custom_info/__manifest__.py

```diff
@@ -7,10 +7,9 @@
     "name": "Base Custom Info",
     "summary": "Add custom field in models",
     "category": "Tools",
-    "version": "14.0.1.0.1",
+    "version": "15.0.1.0.1",
```

Comment:
You can reset the version to `15.0.1.0.0`

Reply:
@Rad0van locally tested, LGTM.
If manifest version is changed as @sebalix sugguests I'll approve it

Reply:
@sebalix could you approve this PR?